devp2p/tests/test_slogging.py

```diff
@@ -1,7 +1,10 @@
 import logging
 import unittest
 from devp2p import slogging
-from StringIO import StringIO
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
So I tried leaving only an `from io import StringIO` which failed the previous build. The reason is that `StringIO.StringIO` accepts both bytes and unicode while `io.StringIO` accepts only unicode.

https://mail.python.org/pipermail/python-list/2013-May/648245.html

I think it makes more sense to try `StringIO.StringIO` than going through all the log calls in the project to change string literals.